### PR TITLE
Dyn 4934 watch node color range impacted

### PR DESCRIPTION
**Pull request by jesusalvino**
### Purpose

Don't display the levels content when the Watch Node is being used for other controls like the Color Range

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

###Reviewers
@QilongTang

###FYIs
@RobertGlobant20 @filipeotero

### CODE REVIEW

**Comment by jesusalvino** [comment]
ColorRange

File: src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs [hunk 1]
```diff
@@ -76,7 +76,16 @@ private void _vm_PropertyChanged(object sender, System.ComponentModel.PropertyCh
                         this.Width = defaultWidthSize;
                     }
                 }
-            }            
+            }
+            else if (e.PropertyName == "IsOneRowContent")
```
Comment (mjkkirschner) [inline src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs:80]: can you use `nameof`?

**Comment by mjkkirschner** [comment]
Would be nice if this could be tested with a UI test, since it has regressed in the past already.

File: src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs (hunk 1 above)
Comment (jesusalvino) [reply src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs]: Done.

**Comment by jesusalvino** [comment]
> Would be nice if this could be tested with a UI test, since it has regressed in the past already.

@QilongTang since I have another update/bug (DYN-4936) related to this WatchNode, Is it a priority? or could be done later if required

**Comment by QilongTang** [comment]
> 

Do you mean you plan to open another PR on the same Jira issue?

**Comment by QilongTang** [comment]
> > > 
> > 
> > 
> > Do you mean you plan to open another PR on the same Jira issue?
> 
> Kindly, I mean focusing on the bugs first and after that take care of the UI Tests (different task) if required.

Please add a unit test under https://github.com/DynamoDS/Dynamo/blob/master/test/DynamoCoreWpfTests/PreviewBubbleTests.cs about this particular edge case

**Comment by jesusalvino** [comment]
imagen

File: test/DynamoCoreWpfTests/PreviewBubbleTests.cs [hunk 2]
```diff
@@ -723,5 +869,11 @@ private void RaiseMouseLeaveNode(IInputElement nv)
 
             DispatcherUtil.DoEvents();
         }
+
+        protected ModelBase GetNode(string guid)
+        {
+            Guid id = Guid.Parse(guid);
+            return ViewModel.Model.CurrentWorkspace.GetModelInternal(id);
```
Comment (reddyashish) [inline test/DynamoCoreWpfTests/PreviewBubbleTests.cs:876]: You can directly return the node model by finding that node in CurrentWorkspace.Nodes property, instead of downcasting it to and converting it to NodeModel again.

File: test/DynamoCoreWpfTests/PreviewBubbleTests.cs [hunk 3]
```diff
@@ -188,9 +190,9 @@ public void PreviewBubble_CodeBlockIsNotInFocus()
             // Click on DragCanvas.
             View.ChildOfType<DragCanvas>().RaiseEvent(
                 new MouseButtonEventArgs(Mouse.PrimaryDevice, 0, MouseButton.Left)
-            {
-                RoutedEvent = Mouse.MouseDownEvent
-            });
+                {
```
Comment (reddyashish) [inline test/DynamoCoreWpfTests/PreviewBubbleTests.cs:193]: Minor: Some alignment issues, with extra spacing and lines.

**Review by reddyashish — COMMENTED** [review]
Looks good to me with couple of comments.

File: test/DynamoCoreWpfTests/PreviewBubbleTests.cs (hunk 2 above)
Comment (jesusalvino) [reply test/DynamoCoreWpfTests/PreviewBubbleTests.cs]: Done.

File: test/DynamoCoreWpfTests/PreviewBubbleTests.cs (hunk 3 above)
Comment (jesusalvino) [reply test/DynamoCoreWpfTests/PreviewBubbleTests.cs]: Done.

**Review by reddyashish — APPROVED** [review]
